examples/layers/JSONLayers/Region.json

```diff
@@ -7,7 +7,8 @@
 	"style"      : "",
 	"projection" : "EPSG:4326",
 	"transparent" : true,
-	"bbox"      : [-61.80983300000002, -21.38963079935857, 55.83665389999998, 51.0890011990783],
+    "unit"      : 1,
```

Comment:
If you want to know the unit of the angles the standard says that you have to use the type of projection. Therefore it's not necessary to add `unit`.